Makefile

```diff
@@ -170,7 +149,7 @@ bdt:
 	# supply this target with the necessary environment vars, e.g.:
 	# make bdt BDT_BASE_URL=<origin of API>
 	docker build --no-cache -t bdt -f Dockerfiles/Dockerfile.bdt .
-	docker run --rm \
+	@docker run --rm \
```

Comment:
BTW. These "@" are added throughout the makefile to prevent echo of possibly sensitive info.